third_party/terraform/resources/resource_container_cluster.go.erb

```diff
@@ -1129,7 +1129,14 @@ func resourceContainerClusterCreate(d *schema.ResourceData, meta interface{}) er
 		}
 	}
 
-	return resourceContainerClusterRead(d, meta)
+	if err := resourceContainerClusterRead(d, meta); err != nil {
+		return err
+	}
+
+	if err := waitForContainerClusterReady(config, project, location, clusterName, d.Timeout(schema.TimeoutCreate)); err != nil {
```

Comment:
Even though we are passing the create timeout here, when the calling operation is Read then the entire Read request will be killed at the read timeout including this call even though the individual call will have a timeout that is longer. In practice this means Read will fail after 2 mins.

Reply:
I think that's okay, but I'm not sure.  Do you think we should do something about that?

Reply:
nvm - I was in the wrong place in code reading this timeout. It's setting the correct timeout.